src/omv/modules/py_tf.c

```diff
@@ -578,9 +483,11 @@ STATIC void py_tf_segment_output_callback(void *callback_data,
 }
 
 STATIC mp_obj_t py_tf_segment(uint n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
-    enum { ARG_roi };
+    enum { ARG_roi, ARG_input_scale, ARG_input_normalization };
```

Comment:
```suggestion
    enum { ARG_roi, ARG_input_scale, ARG_input_stats };
```

Reply:
This is an outdated comment. I changed it previously to what was discussed in the meeting with mean/std.